Snippets/Rabbit/Rabbit_All/ErrorQueue/ErrorQueue.cs

```diff
@@ -28,6 +28,9 @@ public static void ReturnMessageToSourceQueue(IConnection connection, string err
         {
             using (var channel = connection.CreateModel())
             {
+                // Enable publisher confirms so that messages are guaranteed to be sent
```

Comment:
```suggestion
                // Enable publisher confirms so that messages aren't removed from the error queue until the broker confirms it has accepted the new message
```